src/main/kotlin/no/nav/rekrutteringsbistand/stillingssokproxy/SearchClient.kt

```diff
@@ -18,22 +18,31 @@ import org.elasticsearch.client.RestClient
 import org.elasticsearch.client.RestHighLevelClient
 import java.io.IOException
 
-fun sok(jsonbody: String, params: Map<String, List<String>>, indeks: String): SokeResultat {
-    val client = getRestHighLevelClient()
+fun sok(jsonbody: String, params: Map<String, List<String>>, indeks: String): ElasticSearchSvar {
     val request = elasticSearchRequest("GET", "$indeks/_search", params, jsonbody)
+    return gjorRequest(request)
```

Comment:
Er det noen grunn for at vi ikke bruker ø i `gjorRequest` og `sok`?

Reply:
Egentlig ikke, kan endre!